### ISSUE TITLE
can I get the data in the plot directly

### ISSUE THREAD

**Issue by nuaabuaa07** [sfu-db/dataprep#979]
Becase I want to do some postProcess on the anysis result data , can I get the ansys result data directly ? Thinks

**Comment by DiptenduIDEAS** [comment]
This is a very important requirement. Having the capability of exporting the statistics, insights, etc. , in JSON or similar format would be very useful. You might also consider exporting the visualisations in a format such as Vega-Lite.

**Comment by dovahcrow** [comment]
@DiptenduIDEAS Yeah I agree that this is very important. We do have this in design (e.g. `eda.missing.compute_missing`.) but the format is not standardized yet. Having a standardized format will be helpful here I think.
